File: sdks/sdkwork-mail-sdk/sdkwork-mail-sdk-csharp/providers/Sdkwork.Mail.Sdk.Provider.Smtp/src/MailProviderSmtpPackageContract.cs
namespace Sdkwork.Mail.Sdk.Provider.Smtp;

/// <summary>
/// Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
/// </summary>
public static class MailProviderSmtpPackageContract
{
    public const string ProviderKey = "smtp";
    public const string PluginId = "Mail-smtp";
    public const string DriverId = "sdkwork-mail-driver-smtp";
    public const string PackageIdentity = "Sdkwork.Mail.Sdk.Provider.Smtp";
    public const string Status = "future-runtime-bridge-only";
    public const string RuntimeBridgeStatus = "reserved";
    public const bool RootPublic = false;
}

File: sdks/sdkwork-mail-sdk/sdkwork-mail-sdk-flutter/providers/mail_sdk_provider_smtp/lib/src/mail_provider_smtp_package_contract.dart
/// Metadata-only provider package scaffold. No runtime bridge is implemented here yet.
final class MailProviderSmtpPackageContract {
  static const String providerKey = "smtp";
  static const String pluginId = "Mail-smtp";
  static const String driverId = "sdkwork-mail-driver-smtp";
  static const String packageIdentity = "mail_sdk_provider_smtp";
  static const String status = "future-runtime-bridge-only";
  static const String runtimeBridgeStatus = "reserved";
  static const bool rootPublic = false;

  const MailProviderSmtpPackageContract._();
}
